test/iron-menu-behavior.html

```diff
@@ -139,107 +139,129 @@
     </test-fixture>
 
     <script>
+      function waitForIronItemsChanged(selector, callback) {
```

Comment:
Once again, this is about [this thread](https://github.com/PolymerElements/iron-selector/pull/139#discussion_r109078246) in iron-selector. :(